cmd/platform/user.go

```diff
@@ -128,14 +127,14 @@ var searchUserCmd = &cobra.Command{
 }
 
 func init() {
-	userCreateCmd.Flags().String("username", "", "Username")
-	userCreateCmd.Flags().String("email", "", "Email")
-	userCreateCmd.Flags().String("password", "", "Password")
-	userCreateCmd.Flags().String("nickname", "", "Nickname")
-	userCreateCmd.Flags().String("firstname", "", "First Name")
-	userCreateCmd.Flags().String("lastname", "", "Last Name")
-	userCreateCmd.Flags().String("locale", "", "Locale (ex: en, fr)")
-	userCreateCmd.Flags().Bool("system_admin", false, "Make the user a system administrator")
+	userCreateCmd.Flags().String("username", "", "Required. Username for the new user account")
```

Comment:
I think it's a personal preference sort of thing, unless we have a standard for text like this. When printed out, they look like a list, and I think lists look kind of strange with punctuation like that.